etl/models/types/document_tuple_exception_msg.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class DocumentTupleExceptionMsg(str, Enum):
+    """An enum of Exception messages for a DocumentTuple."""
+
+    INVALID_RECORD_TYPE_MSG = "Invalid Record type."
```

Comment:
Nit: if you don't need this stuff except to appease ruff, I'd just disable that ruff warning. It's extra code that doesn't contribute much.